crypto/fipsmodule/bn/montgomery.c

```diff
@@ -119,8 +119,43 @@
 #include <openssl/type_check.h>
 
 #include "internal.h"
+#include "../cpucap/internal.h"
 #include "../../internal.h"
 
+#if !defined(OPENSSL_NO_ASM) &&                                                \
+    (defined(OPENSSL_LINUX) || defined(OPENSSL_APPLE)) &&                      \
+    defined(OPENSSL_AARCH64) && defined(OPENSSL_BN_ASM_MONT)
+
+#include "../../../third_party/s2n-bignum/include/s2n-bignum_aws-lc.h"
+
+#define BN_MONTGOMERY_USE_S2N_BIGNUM 1
```

Comment:
It's a nit. But this is a capability definition. It doesn't mean it's actually used a run-time. Hence the naming is misleading.

Reply:
Would `BN_MONTGOMERY_S2N_BIGNUM_CAPABLE` be ok? Then I will apply it in the next commit.